message.go

```diff
@@ -233,7 +233,7 @@ type MessageSend struct {
 	Content         string                  `json:"content,omitempty"`
 	Embeds          []*MessageEmbed         `json:"embeds"`
 	TTS             bool                    `json:"tts"`
-	Components      []MessageComponent      `json:"components"`
+	Components      []*MessageComponent     `json:"components"`
```

Comment:
`MessageComponent` is an interface. If the array going to consist of pointers, it's going to be extremely inconvenient to pass things in. 
```go
var btn discordgo.MessageComponent = discordgo.Button{}
components := []*discordgo.MessageComponent{
    discordgo.Button{...}, // Not possible
    &btn,
}